File: exercicios em Python/q25.py
import os
print("\n\nEste programa calcula o numero de alunos assima da media.");
n = int(input("\n\nDigite a quantidade de alunos: "))
i = 1
media = 0
while i <= n:
        print("\nDigite a nota do ",i," aluno:")
        nota = int(input(" "))
        if nota >= 7:
                media = media + 1
        i = i + 1

print("\n\nA sala de aula com ",n," alunos, teve ",media," alunos com a nota assima da media.")
print("\n\nObrigado por utilizar nosso programa!")
os.system("pause")

File: exercicios em C/q25.cpp
#include <stdio.h>

int main(void){
	int n, nota, media, i;
	printf("\n\nEste programa calcula o numero de alunos assima da media, levando em conta que a media e de 7.");
	printf("\n\nDigite a quantidade de alunos pertencente a turma: ");
	scanf("%d", &n);
	for (i = 1; i <= n; i++){
		printf("\n\nDigite a nota do %d aluno: ", i);
		scanf("%d", &nota);
		if(nota => 7){
			media = media + 1;	
		}
	}
	printf("\n\nA sala de aula com %d alunos, teve %d alunos com a nota assima da media.", n, media);
	printf("\n\nObrigado por utilizar nosso programa!");
	return 0;
}
